tests/queries/0_stateless/01515_logtrace_function.sql

```diff
@@ -0,0 +1 @@
+SELECT logTrace('message')
```

Comment:
How to test: we can do SYSTEM FLUSH LOGS and find this message in system.text_log.